stdlib/2/typing.pyi

```diff
@@ -9,13 +9,17 @@ from types import CodeType, FrameType, TracebackType
 overload = object()
 Any = object()
 TypeVar = object()
-Generic = object()
-Tuple = object()
-Callable = object()
-Type = object()
 _promote = object()
 no_type_check = object()
-ClassVar = object()
+
+class _SpecialForm:
+    def __getitem__(self, typeargs: Any) -> Any: ...
```

Comment:
I would rather say ``-> object: ...`` otherwise ``Tuple[int].nonexisting(1, 'whatever')`` will be valid.